reactor-netty-core/src/main/java/reactor/netty/util/MapUtils.java

```diff
@@ -0,0 +1,31 @@
+package reactor.netty.util;
+
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * It's a temporary workaround for Java 8 specific performance issue JDK-8161372
+ * and this class should be removed once the Java 8 support is dropped.
```

Comment:
```suggestion
 * and this class should be removed once the Java 8 support is dropped.
 * <p><strong>Note:</strong> This utility class is for internal use only. It can be removed at any time.
```